lib/battle/BattleHex.h

```diff
@@ -8,7 +8,15 @@
  *
  */
 #pragma once
-#include "../GameConstants.h"
+
+namespace BattleSide
+{
+	enum

```

Comment:
ok, np, when you merge it I will change it to BattleSide. Need to change one line I think ;)